app/services/taxonomy/update_taxon.rb

```diff
@@ -23,7 +25,7 @@ def publish(validate: true)
       # so that we can compare the differences between the two versions.
       Taxonomy::SaveTaxonVersion.call(taxon, @version_note)
 
-      Services.publishing_api.put_content(content_id, payload)
+      publishing_api_put_content_request(content_id)
```

Comment:
I wonder if it would be simpler to just copy/paste the previous line and surround with an 'if' block, like we do [here](https://github.com/alphagov/content-tagger/pull/1000/files#diff-63a8e971bcba960dd75982227cab40e0R178). Although normally we'd aim to reduce duplication, in this case I think it's more important to keep the changes minimal and additive, so they're easy to identify.

Reply:
Sadly using an if statement gave me a linting error 😞 `Style/GuardClause: Use a guard clause (return unless brexit_taxon?(taxon_content_id)) instead of wrapping the code inside a conditional expression. (https://rubystyle.guide#no-nested-conditionals)`